<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_titile" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>1440</width>
      <height>98</height>
     </rect>
    </property>
    <widget class="QStackedWidget" name="topStackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1440</width>
       <height>98</height>
      </rect>
     </property>
     <widget class="QWidget" name="page_3"/>
     <widget class="QWidget" name="page_4"/>
    </widget>
   </widget>
   <widget class="QWidget" name="leftWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>1080</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>360</width>
      <height>1080</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_back" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>360</width>
       <height>98</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>53</x>
        <y>36</y>
        <width>110</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgba(102,102,102,1)</string>
      </property>
      <property name="text">
       <string>会议瑞记</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>358</x>
        <y>0</y>
        <width>2</width>
        <height>98</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgba(193, 220, 245, 1);
background-color: rgba(193, 220, 245, 1)</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>360</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(193, 220, 245, 1);
background-color: rgba(193, 220, 245, 1)</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>358</x>
       <y>98</y>
       <width>2</width>
       <height>981</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(193, 220, 245, 1);
background-color: rgba(193, 220, 245, 1)</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>1440</width>
      <height>981</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 251, 255)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>1440</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(193, 220, 245, 1);
background-color: rgba(193, 220, 245, 1)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
